src/Actions/UpdateUserProfileInformation.php

```diff
@@ -23,7 +23,7 @@ class UpdateUserProfileInformation implements UpdatesUserProfileInformation
     public function update($user, array $input)
     {
         Validator::make($input, [
-            'name'  => ['required', 'string', 'max:255', resolve(PoliteUsername::class)],
+            'name'  => ['required', 'string', 'min:3', 'max:30', resolve(PoliteUsername::class)],
```

Comment:
```suggestion
            'name'  => ['required', 'string', 'min:3', 'max:40', resolve(PoliteUsername::class)],
```

this is not the same as username, has 40 on MSQ for display